server/channels/app/channel_bookmark_test.go

```diff
@@ -102,8 +102,12 @@ func TestUpdateBookmark(t *testing.T) {
 			HasPreviewImage: true,
 		}
 
-		th.App.Srv().Store().FileInfo().Save(th.Context, file)
-		defer th.App.Srv().Store().FileInfo().PermanentDelete(th.Context, file.Id)
+		_, appErr := th.App.Srv().Store().FileInfo().Save(th.Context, file)
+		assert.NoError(t, appErr)
```

Comment:
Nit: `Save` return an `error`, hence the variable name should be `err` instead of `appErr`